Applications/ConsumerApi/src/Controllers/Tokens/TokensController.cs

```diff
@@ -79,6 +80,30 @@ public async Task<IActionResult> DeleteToken([FromRoute] string id, Cancellation
         await _mediator.Send(new DeleteTokenCommand { Id = id }, cancellationToken);
         return NoContent();
     }
+
+    [HttpPost("{id}/UpdateContent")]
+    [ProducesResponseType(typeof(HttpResponseEnvelopeResult<UpdateTokenContentResponse>), StatusCodes.Status200OK)]
+    [ProducesError(StatusCodes.Status400BadRequest)]
+    [ProducesError(StatusCodes.Status404NotFound)]
```

Comment:
did you miss 401?

Reply:
These annotations are for the SwaggerUI, which already knows that there might be a 401 because the endpoint requires authorization.

Reply:
ah, nice!